web/server/codechecker_server/permissions.py

```diff
@@ -293,21 +293,51 @@ def __get_perm_record(self, auth_name, is_group):
                 query(SysPerm). \
                 filter(and_(
                     SysPerm.permission == self._perm_name,
-                    SysPerm.name == auth_name,
+                    func.lower(SysPerm.name) == auth_name.lower(),
                     SysPerm.is_group == is_group
                 )).one_or_none()
 
             return record
 
+        def __get_stored_auth_name_and_permissions(self, auth_name, is_group):
+            SysPerm = config_db_model.SystemPermission
+
+            stored_auth_name = auth_name
```

Comment:
Why do we need this variable? It is the same as the parameter `auth_name`. On the caller side we have the value of this variable so there is no need to return back with it.

We can change the function name and the return value of this function to something similar:
```py
def __get_permissions(self, auth_name: str, is_group: bool) -> Set[str]:
    """ TODO: Add a docstring """"
    permissions = set()
    ...

    return permissions
```

Also there is no need to indicate the variable type with the variable names (`permission_set`). It is obvious when I check the variable declaration that it is a set.

Reply:
"stored_auth_name" contains the name as the user first added. It may or may not equal with the "auth_name". The purpose of this function that mine the original auth_name from the database.
Permission set is renamed.